### ISSUE TITLE
[feature] Silent Mob Grinding Detection

### ISSUE THREAD

**Issue by marvinej** [Rocologo/MobHunting#697]
my players hate being spammed about grinding. Can you make a config option to do silent blocking of grinding? Players don't need to be spammed about this.

**Comment by Rocologo** [comment]
Have you tried to clear the text in your lang file? (Dont delete the line - just clear the text)

**Comment by marvinej** [comment]
Yes, i did do this after submitting this suggestion. I suppose that achieves the same result just not as known or user friendly? i was half expecting a stacktrace error or something dumb to happen when i did it.

**Comment by Rocologo** [comment]
Well - some server owners think there is enough settings in my config files and have complained about that, and given me a bad review on SpigotMc

It is hard to give everyone what they want :-)

**Comment by marvinej** [comment]
Ah well they've never seen essentials or infernal mobs i guess.

**Comment by fjeddy** [comment]
I really miss this to, but I don't want it silenced, I would like a cooldown before the next message is sent.

"You are grinding, no more cash from you" --> Cooldown for 30 seconds starts, for every mob that is killed before the cooldown runs out, then the timer resets back to 30 seconds. So they will only get a new message if 30 seconds pass without killing a mob.

The spam is highly annoying when playing OneBlock or you're at an XP farm or similar.
